<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBrowser</class>
 <widget class="QWidget" name="ToolBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#btnSave,#btnLoad,#btnPreferences,#btnAbout {
 text-align:center;
 height: 25px;
 background-color: none;
 border:none;
}

#btnSave:hover,
#btnLoad:hover,
#btnPreferences:hover,
#btnAbout:hover,
#btnCollapse:hover{
 background-color: rgba(0, 0, 0, 60);
 border: 1px solid rgba(0, 0, 0, 30);
 border-radius:5px;
}

#btnSave {
 qproperty-icon: url(&quot;:/gui/icons/scopy-default/icons/save.svg&quot;);
}
#btnLoad {
 qproperty-icon: url(&quot;:/gui/icons/scopy-default/icons/load.svg&quot;);
}
#btnPreferences {
 qproperty-icon: url(&quot;:/gui/icons/scopy-default/icons/preferences.svg&quot;);
}
#btnAbout {
 qproperty-icon: url(&quot;:/gui/icons/scopy-default/icons/info.svg&quot;);
}
#logo {
background-image: url(:/gui/icons/scopy-default/icons/logo_analog.svg);
background-repeat: no-repeat;
background-position: center center;
background-repeat: no-repeat;
min-height:48px;
}

#btnCollapse {
 height:40px;
 background-image: url(:/gui/icons/scopy-default/icons/logo.svg);
 background-repeat: no-repeat;
 background-position: center center;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="btnCollapse">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="homePlaceholder" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="scopy::ToolMenu" name="wToolMenu">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>398</width>
        <height>16</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="subsection_line" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="saveLoadGroup" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLoad">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnPreferences">
     <property name="text">
      <string>Preferences</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnAbout">
     <property name="text">
      <string>About</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="logo">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>scopy::ToolMenu</class>
   <extends>QWidget</extends>
   <header>toolmenu.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
